src/database/migrations/20211018084932_create_table_users.ts

```diff
@@ -6,9 +6,12 @@ export async function up (knex: Knex): Promise<void> {
       return knex.schema.createTable('users', function (table) {
         table.uuid('id').primary()
         table.string('name', 100).notNullable()
-        table.text('description').notNullable()
         table.string('avatar').notNullable()
-        table.string('type', 15).notNullable()
+        table.string('email', 150).notNullable()
+        table.string('password', 72)
```

Comment:
kenapa password di batasi lebar datanya 72?

Reply:
eh 1 lagi ,, walaun pasword pasti bakalan ke isi,, kalo boleh tetp pake not nullable y

Reply:
ini ada kemungkinan null ketika signup with google kang

Reply:
72 berdasarkan max length dari bcrypt kang

Reply:
ref: https://cheatsheetseries.owasp.org/cheatsheets/Password_Storage_Cheat_Sheet.html

Reply:
oooo bgitu ya kondisi nya,, isee 👍